themes/mz_hugo/layouts/partials/footer.html

```diff
@@ -1,3 +1,35 @@
+<div class="footer__partners-wrapper">
+
+  <div class="footer__partners">
+
+    <div class="footer__partners-left">
+      <h1 class="footer__partners-title">Curated by</h1>
+
+      <a class="footer__partners__link" target="_blank" href="http://www.fesagency.pt/">
+        <img src="images/partners/fes-logo.svg">
```

Comment:
You need an absolute path or else it won't work on all pages. Check for example: https://58b80755a7e2d26a2eeff88b--pwit-cms.netlify.com/mission/

You need on all these something like:
`<img src="/images/partners/fes-logo.svg">`